### PR TITLE
Sane queue

### PR DESCRIPTION
**Pull request by AdityaAtulTewari**

### CODE REVIEW

**Review by patrickkenney9801 — APPROVED** [review]
Good idea

File: pando-rt/src/global_ptr.cpp [hunk 1]
```diff
@@ -79,7 +79,7 @@ void* asNativePtr(GlobalAddress globalAddr) noexcept {
 #if defined(PANDO_RT_USE_BACKEND_PREP)
 
   // yield to other hart and then issue the operation
-  hartYield();
+  //hartYield();
```
Comment (patrickkenney9801) [inline pando-rt/src/global_ptr.cpp:82]: assuming these were needed due to the subpar queue

Comment (AdityaAtulTewari) [reply pando-rt/src/global_ptr.cpp:82]: That is my hypothesis and it seems to be borne out, there is another yield that exists after calling executeOn. Removing that seems to torpedo perf. I can't figure out why.

Comment (patrickkenney9801) [reply pando-rt/src/global_ptr.cpp:82]: strange stuff

**Review by patrickkenney9801 — APPROVED** [review]

**Pull request merged**